eng/common/scripts/Submit-PullRequest.ps1

```diff
@@ -47,6 +50,33 @@ $headers = @{
 
 $query = "state=open&head=${PROwner}:${PRBranch}&base=${BaseBranch}"
 
+function AddLabels([int] $prNumber)
```

Comment:
If you are going to make this a function I would also take the label list as a parameter instead of relying on the global variable.

Reply:
Changed to have more parameter on function.